lib/config/config.js

```diff
@@ -112,6 +117,20 @@ function load (app) {
   require('./production')(app)
 }
 
+function checkPackageVersion (name, pkg) {
+  const expected = pkg.dependencies[name]
+  const installed = require(`../../node_modules/${name}/package.json`)
+
+  if (!semver.satisfies(installed.version, expected)) {
+    console.error(
```

Comment:
Is the only result of a failing check a line in the log? Should we have something very visible in the admin ui as well?

Reply:
Below I `process.exit(1)`

Reply:
Oops that you do.